rc/parinfer.kak

```diff
@@ -1,37 +1,77 @@
-declare-option -docstring %{Whether to automatically update the buffer on changes} bool parinfer_enabled yes
+define-command -docstring "parinfer-enable-window [<mode>]: enable Parinfer for current window.
+Modes:
+    -indent  Preserve indentation and fix parentheses (default).
+    -paren   Preserve parentheses and fix indentation.
+    -smart   Try to be smart about what to fix." \
+parinfer-enable-window -params ..1 %{
+    require-module parinfer
+    try %{
+        parinfer -paren
+        # if parinfer -paren fails, we don't set parinfer_enable to true
+        # thus making sure that hooks do not break our code
+        set-option window parinfer_enabled true
+    } catch %{
+        # set up recovery hooks that will re-enable parinfer when parens are balanced
+        hook -group parinfer-try-paren window NormalIdle .* parinfer-try-paren
+        hook -group parinfer-try-paren window InsertIdle .* parinfer-try-paren
+        echo -markup "{Error}parinfer error: %val{error}"
+        echo -debug "parinfer error: %val{error}"
+    }
+    evaluate-commands %sh{
+        mode="${1:-indent}"
```

Comment:
The default mode should be `-smart`

Reply:
The default one was `-indent` and it still is: https://github.com/eraserhd/parinfer-rust/blob/master/rc/parinfer.kak#L24

Should I change it to `-smart` everywhere?

Reply:
I understood that `parinfer` should use `indent` by default, and `parinfer-enable-window` should use `smart` by default